zerver/openapi/zulip.yaml

```
@@ -13592,12 +13592,23 @@ paths:
 
                           The fake email domain that will be used for new bots created this
                           organization. Useful for UI for creating bots.
-                      realm_uri:
+                      realm_url:
                         type: string
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
                           The URL for the organization.
+                          **Changes**: New in Zulip 8.0 (feature level 192). The name `realm_url`
+                          is preferred use than `realm_uri` as later is deprecated.
```

"The name `realm_url` is preferred over `realm_uri` as the latter is deprecated."

Though actually perhaps better:
```suggestion
                          **Changes**: New in Zulip 8.0 (feature level 192), replacing the deprecated `realm_uri`.
```